dev/crypto.md

```diff
@@ -95,6 +95,38 @@ random seed.
 Algorand uses the [ed25519][ed25519] digital signature scheme to sign
 data.
 
+Algorand changes the ed25519 verification algorithm in the following way:
```

Comment:
Should we add a citation to https://eprint.iacr.org/2020/1244.pdf in order to justify these changes? I think this would make things clearer in the future.